atakama/rule_engine.py

```diff
@@ -0,0 +1,195 @@
+"""Atakama keyserver ruleset library"""
+import abc
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Dict, Union, TYPE_CHECKING
+import logging
+
+import yaml
+
+from atakama import Plugin
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+class RequestType(Enum):
+    DECRYPT = "decrypt"
+    SEARCH = "search"
+    CREATE_PROFILE = "create_profile"
+    ACTIVATE_LOCATION = "activate_location"
+    CREATE_LOCATION = "create_location"
+    RENAME_FILE = "rename_file"
```

Comment:
```suggestion
    RENAME = "rename"
```
As with `decrypt` I think it is ideal to avoid file-based terminology at this layer

Reply:
right, removed